scripts/profiler/merge_data.py

```diff
@@ -9,40 +9,48 @@
 
 
 def main():
-    descr = "Merge data from device and dongle. ble_peer_events should be" \
-            " registered at the beginning and at the end of measurements"  \
-            " (used to compensate clock drift)."
+    descr = "Merge data from peripheral and central. Synchronization events" \
```

Comment:
Capitalization inconsistency with profiler.rst's line 121.

Reply:
ok, I will create new PR for this one (since this one is already closed)